<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaInfoServicio</class>
 <widget class="QDialog" name="ventanaInfoServicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>605</width>
    <height>453</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>605</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Información del servicio</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../recursos/qtRecursosIconos.qrc">
    <normaloff>:/iconos/iconos/icono-app-dgtic.png</normaloff>:/iconos/iconos/icono-app-dgtic.png</iconset>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">#ventanaInfoServicio {
	background-color: #fff;
}

#frameGeneralInfoServicio .QPushButton{
	color: #ffffff;
	background-color: #417fd2;
	border: 1px solid rgb(0, 0, 0);
	border-radius: 6px;
	padding-bottom: 2px;
	font: 8.5pt &quot;Segoe UI Black&quot;;
}

#frameGeneralInfoServicio .QPushButton::hover{
	background-color: #5c8bc9;
}

#frameGeneralInfoServicio .QPushButton::pressed{
	background-color: #063c85;
}</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frameGeneralInfoServicio">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frameCamposInfoServicio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>570</width>
        <height>391</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>570</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>451</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vlCamposInfoServicio">
        <item>
         <layout class="QHBoxLayout" name="hlFechaServicio">
          <item>
           <widget class="QLabel" name="labelInfoFechaServicio">
            <property name="text">
             <string>Fecha del servicio:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="deInfoFechaServicio">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hlNombreDepartamento">
          <item>
           <widget class="QLabel" name="labelInfoNombreDepartamento">
            <property name="text">
             <string>Nombre del departamento:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hsNombreDepartamento">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>17</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="inputInfoNombreDepartamento"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hlFallaPresenta">
          <item>
           <widget class="QLabel" name="labelInfoFallaPresenta">
            <property name="text">
             <string>Falla que presenta:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hsFallaPresenta">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="inputInfoFallaPresenta"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hlServicioPrestado">
          <item>
           <widget class="QLabel" name="labelInfoServicioPrestado">
            <property name="text">
             <string>Servicio prestado:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hsServicioPrestado">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>67</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="inputInfoServicioPrestado"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hlNombresTecnicos">
          <item>
           <widget class="QLabel" name="labelInfoNombresTecnicos">
            <property name="text">
             <string>Nombre de los técnicos:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hsNombreTecnicos">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>33</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="inputInfoNombresTecnicos"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="vlInfoObservacionesAdicionales">
          <item>
           <widget class="QLabel" name="labelInfoObservacionesAdicionales">
            <property name="text">
             <string>Observaciones adicionales:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="teInfoObservacionesAdicionales"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="frameBotonesAccionServicio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>570</width>
        <height>101</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>570</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>371</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="hlBotonesAccionServicio">
        <item>
         <widget class="QPushButton" name="botonAcualizarInfoServicio">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Actualizar info</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="botonEliminarServicio">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Eliminar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="botonCancelar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Cancelar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../recursos/qtRecursosIconos.qrc"/>
 </resources>
 <connections/>
</ui>
